packages/api/src/entities/User.ts

```diff
@@ -10,9 +10,38 @@ export class User extends Node<User> {
   @Property({ columnType: 'text' })
   name: string;
 
-  constructor({ name, ...extraValues }: UserConstructorValues) {
+  @Property({ columnType: 'text' })
+  pronouns: string;
+
+  @Property({ columnType: 'text' })
+  location?: string;
+
+  @Property( { columnType: 'boolean'})
+  hireable?: boolean;
+
+  @Property({ columnType: 'text' })
+  purpose: string;
+
+  @Property({ columnType: 'text' })
+  schoolName:string;
+
+  @Property({ columnType: 'text' })
+  major: string;
+
+  @Property({ columnType: 'Date' })
+  graduationDate?: Date;
```

Comment:
Optional properties need a `nullable` option.

`hireable` probably shouldn't be optional since that'll be a checkbox on the UI

I can't imagine a scenario where they would have a school and major and not a graduation date. Either it'll be in the future or past, but there should be one. If they are in highschool then they could leave all blank since we're only looking for college.

```suggestion
  @Property({ columnType: 'text', nullable: true })
  location?: string;

  @Property( { columnType: 'boolean' })
  hireable: boolean;

  @Property({ columnType: 'text' })
  purpose: string;

  @Property({ columnType: 'text', nullable: true })
  schoolName?: string;

  @Property({ columnType: 'text', nullable: true })
  major?: string;

  @Property({ columnType: 'Date', nullable: true })
  graduationDate?: Date;
```